client/src/components/DownloadModal.js

```diff
@@ -45,7 +51,17 @@ export const DownloadModal = ({
 
   return (
     <>
-      {icon ? (
+      {icon && sampleMetadataOnly ? (
+        <Anchor
+          icon={icon}
+          onClick={handleClick}
+          disabled={disabled}
+          label={
+            sampleMetadataOnly ? <Text color="brand">{buttonLabel}</Text> : ''
```

Comment:
Yes, this is always true. This is to prevent the `buttonLabel` from being rendered on the download sample icon button in the project samples table (pls see the screenshot below).

![Screenshot 2024-06-13 at 1 21 56 PM](https://github.com/AlexsLemonade/scpca-portal/assets/31800566/4445a829-284d-419c-b3b7-d25c125566ab)